homeassistant/components/lgthinq/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "domain": "lgthinq",
+  "name": "LG ThinQ",
+  "codeowners": ["@LG-ThinQ-Integration"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/lgthinq/",
+  "iot_class": "cloud_push",
+  "loggers": ["pycountry", "thinqconnect"],
+  "requirements": [    
+    "pycountry==23.12.11",
+    "thinqconnect==0.9.2"
```

Comment:
Same comment as before: https://github.com/home-assistant/core/pull/123341/files#r1711423028

I could not find the public / open-source repository providing this package. pypi doesn't have any public repository linked, and couldn't find it with a search on GitHub either.

Reply:
Please refer to the link below.
- https://pypi.org/project/thinqconnect/

Reply:
But where is it on GitHub?

Reply:
@joostlek 
We are planning to open the repository soon.

Reply:
We are planning to release it in October.

Reply:
Let me share this with someone in charge.
In that case, I think  we can reschedule the release.